index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>HTML Mastery Quiz</title>
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+	<div id="wrapper">
+		<article id="card">
+			<div class="section1">
+				<header>
+				<h1>Yellow Dunk Cards</h1>
+				</header>
+				<section>
+					<div class="image" style="width:100px;height:100px;background-color:yellow;"></div>
+				</section>
+				<section>
+					<p>Bacon ipsum dolor amet landjaeger leberkas pork loin picanha spare ribs. Frankfurter leberkas flank pork loin kevin cow strip steak pig turkey beef ribs. Pork loin picanha shank chicken, tongue meatloaf pork chop. Corned beef swine short loin alcatra biltong kielbasa tongue. Flank frankfurter ball tip bresaola corned beef filet mignon ground round pastrami. Picanha salami jowl kevin.</p>
+				</section>
+				<section>
+					<h4 class="avail">Available</h4>
+				</section>
+			</div>
+			<div class="section2">
+				<header>
+					<h3>Specifications</h3>
+				</header>
+				<section>
+					<ul>
+						<li>Size: 4" x 4"</li>
+						<li>Weight: 8 oz.</li>
+					</ul>
+				</section>
+				<footer>
+					<p class="footer">Specifications valid until 04/01/2016</p>
+				</footer>
+			</div>
+			<div class="section3">
+				<header>
+					<h3>Pricing</h3>
+				</header>
+				<section>
+					<ul>
+						<li>1: $1.99</li>
+						<li>2-50: $1.49</li>
+						<li>51+: $0.99</li>
+					</ul>
+				</section>
+			</div>
+		</article>
+		<article id="card">
```

Comment:
IDs must be unique per page. You are using `card` multiple times.  Just change all instances to classes and you'll be OK. Make sure to update your CSS to reflect the switch from `id` to `class`